lib/Categories/Php.php

```diff
@@ -69,6 +69,7 @@ public function getData() {
 			'memory_limit' => $this->phpIni->getBytes('memory_limit'),
 			'max_execution_time' => $this->phpIni->getNumeric('max_execution_time'),
 			'upload_max_filesize' => $this->phpIni->getBytes('upload_max_filesize'),
+			'loaded_extensions' => implode(',', get_loaded_extensions() ?? [])
```

Comment:
Did you check with a local version, that the survey_server app doesn't crash with the value missing/being there?
Also you need to add it to https://github.com/nextcloud/survey_server/blob/master/data.json

Reply:
Not yet :see_no_evil: Thanks for explaining :+1: